orchard-core/src/main/scala/com/salesforce/mce/orchard/system/actor/ResourceMgr.scala

```diff
@@ -167,6 +180,17 @@ object ResourceMgr {
         ps.ctx.log.info(s"${ps.ctx.self} (running) received GetResourceInstSpec($replyTo)")
         resourceInst ! ResourceInstance.GetResourceInstSpec(replyTo)
         Behaviors.same
+      case CreateResourceInst(replyTo, instId) =>
```

Comment:
why would resource in `running` state also be able to create new instances?

Reply:
good point, updated